CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Unreleased features
+
+(None yet)
+
+# 0.3.0 (2024-08-13)
+
+New features:
+
+- Integrated libp2p for transport encryption and authentication of peers.
+- Added a password protection – peers must specify the same `--secret` to connect.
+- You can now provide default startup arguments in `.ethersync/config`.
+- When editors send
+
+Bug fixes:
+
+- The daemon is now more resilient against misbehaving peers and editors, and won't crash as easily.
+
+# 0.2.2 (2024-08-01)
+
+New features:
+
+- Persist the CRDT to `.ethersync/doc`, so that peers can run independently, and sync back later.
+- When restarting the daemon, calculate file diffs since the daemon was last online, and apply them to the CRDT.
+- Dynamically create files when they are opened.
+- When files are deleted, delete them for all peers.
+- Sandbox file I/O to be restricted into the shared project folder.
+
+# 0.2.1 (2024-07-26)
+
+New features:
+
+- Share multiple files per directory.
+- Transmit cursor positions, and display them in the Neovim plugin.
```

Comment:
Even though not a core feature, we could mention adding a nix flake here?

Reply:
Also it seems we have done some improvements to the daemon-editor protocol specification (it might be good practice to actively mention protocol tweaks and changes in the dev log until we have an actual versioning of that protocol)

See 23246a3

Reply:
> Should we make sure that a release exists for each of those tags?

I think probably not – because we didn't update the version in the Cargo.toml, it will show the wrong version. Except if we start little release branches, and hotfix that there…

I'd be fine with updating the version from now on, when we do the next release! Which could be soon! :)

Reply:
> protocol tweaks and changes

Oh yeah! We dropped the revisioning for the "cursor" messages from the protocol at some point! That's one of the changes I can think of, in addition to supporting JSON-RPC requests now.